backend_new/prisma/schema.prisma

```diff
@@ -0,0 +1,1025 @@
+  unitGroup                    UnitGroup?                       @relation(fields: [unitGroupId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+  amiChart                     AmiChart?                        @relation(fields: [amiChartId], references: [id], onDelete: NoAction, onUpdate: NoAction)
+
+  @@map("unit_group_ami_levels")
+}
+
+model UserPreferences {
+  sendEmailNotifications Boolean      @default(false) @map("send_email_notifications")
+  sendSmsNotifications   Boolean      @default(false) @map("send_sms_notifications")
+  userId                 String       @id() @unique() @map("user_id") @db.Uuid
+  favoriteIds            String[]     @default([]) @map("favorite_ids")
+  userAccounts           UserAccounts @relation(fields: [userId], references: [id], onDelete: Cascade, onUpdate: NoAction)
+  favoritesListings      listings[]
+
+  @@map("user_preferences")
+}
+
+// END DETROIT SPECIFIC
+
+enum ApplicationMethodsTypeEnum {
+  Internal
+  FileDownload
+  ExternalLink
+  PaperPickup
+  POBox
+  LeasingAgent
+  Referral
+
+  @@map("application_methods_type_enum")
+}
+
+enum LanguagesEnum {
+  en
+  es
+  vi
+  zh
+  tl
+
+  @@map("languages_enum")
+}
+
+enum ListingEventsTypeEnum {
+  openHouse
+  publicLottery
+  lotteryResults
+
+  @@map("listing_events_type_enum")
+}
+
+enum ApplicationAddressTypeEnum {
+  leasingAgent
+
+  @@map("listings_application_address_type_enum")
+}
+
+enum ReviewOrderTypeEnum {
+  lottery
+  firstComeFirstServe
+  waitlist
+
+  @@map("listings_review_order_type_enum")
+}
+
+enum ListingsStatusEnum {
+  active
+  pending
+  closed
+
+  @@map("listings_status_enum")
+}
+
+enum MultiselectQuestionsApplicationSectionEnum {
+  programs
+  preferences
+
+  @@map("multiselect_questions_application_section_enum")
+}
+
+enum YesNoEnum {
+  yes
+  no
+
+  @@map("yes_no_enum")
+}
+
+enum RuleEnum {
+  nameAndDOB
+  email
+
+  @@map("rule_enum")
+}
+
+enum FlaggedSetStatusEnum {
+  flagged
+  pending
+  resolved
+
+  @@map("flagged_set_status_enum")
+}
+
+enum IncomePeriodEnum {
+  perMonth
+  perYear
+
+  @@map("income_period_enum")
+}
+
+enum ApplicationStatusEnum {
+  draft
+  submitted
+  removed
+
+  @@map("application_status_enum")
+}
+
+enum ApplicationSubmissionTypeEnum {
+  paper
+  electronical
+
+  @@map("application_submission_type_enum")
+}
+
+enum ApplicationReviewStatusEnum {
+  pending
+  pendingAndValid
+  valid
+  duplicate
+
+  @@map("application_review_status_enum")
+}
+
+enum UnitsStatusEnum {
+  unknown
+  available
+  occupied
+  unavailable
+
+  @@map("units_status_enum")
+}
+
+enum HomeTypeEnum {
+  apartment
+  duplex
+  house
+  townhome
+
+  @@map("listings_home_type_enum")
+}
+
+enum MarketingSeasonEnum {
+  spring
+  summer
+  fall
+  winter
+
+  @@map("listings_marketing_season_enum")
+}
+
+enum MarketingTypeEnum {
+  marketing
+  comingSoon
+
+  @@map("listings_marketing_type_enum")
+}
+
+enum RegionEnum {
+  Greater_Downtown
```

Comment:
This is super Detroit specific, makes me wonder if it should be in the data model this explicitly but idk

Reply:
yeah I think eventually we'll want this to be a table and a relationship instead of a enum (like for sass this doesn't make sense as an enum)

but that's a bridge I think we cross as a 2nd pass through this